pkg/asset/manifests/vsphere/infrastructure.go

```diff
@@ -103,15 +106,33 @@ func GetInfraPlatformSpec(ic *installconfig.InstallConfig, clusterID string) *co
 	return &platformSpec
 }
 
-// vipsToCIDRs takes a single ip address and converts it to CIDR notation.
-func vipsToCIDRs(vips []string) []configv1.CIDR {
-	cidrs := make([]configv1.CIDR, len(vips))
-	for i, vip := range vips {
-		mask := "/32"
-		if utilsnet.IsIPv6String(vip) {
-			mask = "/128"
+// vipsToCIDR takes a single ip address and converts it to CIDR notation.
+func vipToCIDR(vip string) configv1.CIDR {
+	mask := "/32"
+	if utilsnet.IsIPv6String(vip) {
+		mask = "/128"
+	}
+	return configv1.CIDR(vip + mask)
+}
+
+func appendCIDRIfMissing(slice []configv1.CIDR, item configv1.CIDR) []configv1.CIDR {
+	for _, existing := range slice {
+		if existing == item {
+			return slice
 		}
-		cidrs[i] = configv1.CIDR(vip + mask)
 	}
-	return cidrs
+	return append(slice, item)
+}
+
+func getInstallConfigVIPs(ic *installconfig.InstallConfig) []string {
+	vips := make([]string, 0, len(ic.Config.VSphere.APIVIPs)+len(ic.Config.VSphere.IngressVIPs))
+	for _, vip := range ic.Config.VSphere.APIVIPs {
+		vips = append(vips, vip)
+	}
+
+	for _, vip := range ic.Config.VSphere.IngressVIPs {
+		vips = append(vips, vip)
+	}
+
+	return vips
 }
```

Comment:
```suggestion
// getVIPsFromInstallConfig returns VIPs from the install config.
func getVIPsFromInstallConfig(ic *installconfig.InstallConfig) []string {
	vips := make([]string, 0, len(ic.Config.VSphere.APIVIPs)+len(ic.Config.VSphere.IngressVIPs))
	vips = append(vips, ic.Config.VSphere.APIVIPs...)
	vips = append(vips, ic.Config.VSphere.IngressVIPs...)
	return vips
}
```

nit: Cleaner and go-lint is happy :smile: